index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | Home</title>
+    <!-- Google Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
+    <!-- Boxicon -->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <!-- CSS -->
+    <link rel="stylesheet" href="styles.css">
+
+<body>
+    <!-- Start of Header Container -->
+    <header>
+        <nav class="navbar">
+            <div class="container flex">
+                <a href="index.html" class="logo">Evans</a>
+                <ul class="menu">
+                    <li><a href="#portfolio">Portfolio</a></li>
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                </ul>
+                <i class='bx bx-menu'></i>
+            </div>
+        </nav>
+    </header>
+    <!-- End of header container -->
+
+    <!-- start of Main -->
+    <main class="banner">
+        <div class="container">
```

Comment:
- - You have done great, your project looks like the Figma design. :+1: But there are some changes you need to make before moving to the next milestone. :smile: 
- [ ] Following the best practices for the use of semantic tags, this section should be wrapped in a `section` tag. the `section tag` represents a generic standalone section of the document, so I'll suggest you use it here. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. :smile: :wave: